Assets/Scripts/Edges/EdgeGameObjectFactory.cs

```diff
@@ -50,5 +50,27 @@ void CreateNewEdgeGameObjectFor(Edge edge, IList<Vector3> edgePoints)
       lineRenderer.positionCount = positionCount;
       return (line, lineRenderer);
     }
+
+    private void GenerateCollider(GameObject line, Vector3 firstEdgePoint, Vector3 secondEdgePoint)
+    {
+      var colliderGameObject = new GameObject(Constants.ColliderGameObjectName);
+      colliderGameObject.transform.parent = line.transform;
+      var capsule = colliderGameObject.AddComponent<CapsuleCollider>();
+      SetColliderProperties(colliderGameObject, firstEdgePoint, secondEdgePoint);
```

Comment:
You might as well pass capsule instead of colliderGameObject and extracting capsule again anyway